runtime/compiler/control/rossa.cpp

```diff
@@ -1586,6 +1586,14 @@ onLoadInternal(
       ((TR_JitPrivateConfig*)(jitConfig->privateConfig))->iProfiler = NULL;
       }
 
+#if defined(J9VM_OPT_JITSERVER)
+   // JITServer doesn't support JProfiling
+   if (persistentMemory->getPersistentInfo()->getRemoteCompilationMode() == JITServer::CLIENT
+       || persistentMemory->getPersistentInfo()->getRemoteCompilationMode() == JITServer::SERVER)
+      {
+      TR::Options::getCmdLineOptions()->setOption(TR_DisableJProfilerThread);
```

Comment:
I would use do this in J9Options.cpp in `J9::Options::setupJITServerOptions()`

Also add  `self()->setOption(TR_EnableJProfiling, false);` as I don't see it.